streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -931,7 +930,7 @@ public synchronized Pattern latestResetTopicsPattern() {
      * NOTE this function would not needed by developers working with the processor APIs, but only used
      * for the high-level DSL parsing functionalities.
      */
-    public SubscriptionUpdates subscriptionUpdates() {
+    public InternalTopologyBuilder.SubscriptionUpdates subscriptionUpdates() {
```

Comment:
This is a public interface change right? Should it be in the KIP?

Reply:
That's a good point... I moved this class from `StreamPartitionAssignor` to `InternalTopologyBuilder` since I think it is better belonging to the other class. But for compatibility I'll have to keep it as is I think..